### ISSUE TITLE
Proposal: Add "ClearAll" method to TreeView control

### ISSUE THREAD

**Issue by dan-meier** [microsoft/microsoft-ui-xaml#3547]
This method would DEselect all nodes in the tree -- the analog opposite of the existing SelectAll method.

Currently, clearing all checked nodes is unintuitive. Creating a straightforward method that matches the simplicity of the SelectAll method would reduce developer friction, particularly for those new to the control.

**Comment by StephenLPeters** [comment]
There is the SelectedNodes and a selectedItems property on treeview, you can just call clear on those collections right?

**Comment by dan-meier** [comment]
@StephenLPeters   Yes...but it took a bloody long time to figure that out!  Given there are matching ExpandAll/CollapseAll methods, it makes intuitive sense to add a ClearAll method to the existing SelectAll, even if it's just a wrapper over SelectedNodes.Clear().

**Comment by StephenLPeters** [comment]
Hmm, I see... I think it probably makes sense to have a clear all if there is a SelectAll as well. @predavid-zz

**Comment by fabianoriccardi** [comment]
I agree that it would be nice having the opposite of SelectAll, whatever is its name.

The same can be applied to other similar controls which have *SelectAll* but lacks of unselection method, such as ListView and ListBox.

Looking at WPF, such method exists and its name is *UnselectAll*.
